stockflux-launcher/src/App.js

```diff
@@ -1,40 +1,56 @@
 import React from 'react';
 import cx from 'classnames';
-import "stockflux-components";
+import 'stockflux-components';
 import { WindowHooks, Constants } from 'openfin-react-hooks';
-import { FaChevronUp, FaChevronLeft, FaChevronRight, FaCog } from 'react-icons/fa';
+import {
+  FaChevronUp,
+  FaChevronLeft,
+  FaChevronRight,
+  FaCog
+} from 'react-icons/fa';
 import AppShortcuts from './AppShortcuts';
-import FreeTextSearch from './FreeTextSearch';
+import FreeTextSearch from './free-text-search/FreeTextSearch';
 import ToolBar from './ToolBar';
 import './App.css';
 
 export default () => {
   const [windowState, windowActions] = WindowHooks.useCurrentWindowState();
 
+  console.log('screen edge', windowState);
+  console.log('windowActions', windowActions);
+
   return (
     <div className={cx('app', windowState.edge)}>
       <AppShortcuts />
-      <FreeTextSearch />
-      <ToolBar tools={[
-        {
-          label: <FaCog />
-        },
-        {
-          label: <FaChevronUp />,
-          onClick: windowActions.lockTop,
-          disabled: () => windowState.edge === Constants.ScreenEdge.TOP
-        },
-        {
-          label: <FaChevronLeft />,
-          onClick: windowActions.lockLeft,
-          disabled: () => windowState.edge === Constants.ScreenEdge.LEFT
-        },
-        {
-          label: <FaChevronRight />,
-          onClick: windowActions.lockRight,
-          disabled: () => windowState.edge === Constants.ScreenEdge.RIGHT
-        }
-      ]} />
+      <FreeTextSearch dockedTo={windowState.edge} />
+      <ToolBar
+        tools={[
+          {
+            label: <FaCog />
+          },
+          {
+            label: <FaChevronUp />,
+            onClick: () => {
+              windowActions.lockTop();
+            },
+            disabled: () => windowState.edge === Constants.ScreenEdge.TOP
```

Comment:
Any reason why these are arrow functions rather than booleans?

Reply:
No idea, it works just fine with a boolean. I did not really dive into this particular piece of code, was focused more on the child pop-up window. But a very good spot, thanks! Fixed -> c5bba2ac17e81d4069a1c54bc0a431f676deb99b